docs/source/trustme-trio-example.py

```diff
@@ -1,11 +1,15 @@
 # trustme-trio-example.py
 
+import ssl
+
 import trustme
 import trio
 
 # Create our fake certificates
 ca = trustme.CA()
+client_ca = trustme.CA()
```

Comment:
I feel like for a simple "here's how it works" example, it might be better to use a single CA for both certs? Of course a more complicated test setup might want to create multiple CAs and configure trust appropriately, but even if you're starting with an example that doesn't do that, it should be obvious enough how to modify it, right?

Reply:
Well, one CA could be used. It's just that var naming help around line 24 where it says that "client CA should be trusted by server context" which I think makes it a bit more explicit.

Reply:
I can remove it if you insist

Reply:
@njsmith so how do we proceed with this?

Reply:
I see what you're saying about making clear which context is using the CA for what purpose, but I think it'd be simpler and just as clear to use a single CA, and then make the comments say:

```python
    # Set up the server's SSLContext to trust our fake CA,
    # so it can validate the client's cert.
    ca.configure_trust(server_ssl_context) 
```

```python
    # Set up the client's SSLContext to trust our fake CA,
    # so it can validate the server's cert.
    ca.configure_trust(client_ssl_context) 
```

Reply:
Agreed.